### DISCUSSION TITLE
UV modifications in trenchbroom do not show in godot

### DISCUSSION THREAD

**Discussion by al0fdf** [func-godot/func_godot_plugin#69] [Q&A] [answered]
In my project, I have 3 textures: floor, wall and bed. I have left the brushes using the floor and wall as is, but needed some adjustments to get the bed texture to position properly.

Here is the level in trenchbroom before the UV adjustements:
![image](https://github.com/user-attachments/assets/7bca5eff-d1db-4f98-9f0d-5beb91fb8941)

and here is the level after:

![image](https://github.com/user-attachments/assets/c001b267-11d2-4a0a-9145-d862ca1d8276)

The textures show properly in trenchbroom, however when importing into godot, the UV transformations I performed do not show up, and the bed texture is treated as a default tiling one:
![image](https://github.com/user-attachments/assets/7ed17b72-7c40-435b-8983-2077e566136e)

Are there any settings I need to change to have the texture in godot look the same as in trenchbroom?

**Comment by RhapsodyInGeek** [comment]
Try using a square texture, not a rectangular one.

**Reply by al0fdf** [reply]
I've changed the 96x128 texture I used for the bed to a 128x128, however it still shows in godot as a repeated texture.

![image](https://github.com/user-attachments/assets/d589d085-4c78-4508-bdc5-5a04dfe3a978)
![image](https://github.com/user-attachments/assets/ee99166b-0358-4aa9-a08c-bb88f43ac67a)
![image](https://github.com/user-attachments/assets/878ccd11-2ccb-445b-a92e-36cd51e4e170)

**Reply by RhapsodyInGeek** [reply]
You may need to delete the material that was auto-generated when you first built it. Hard to say though. I don't really have the time to troubleshoot this personally, but you may get more help more quickly by asking in the func-godot-help channel on [the official Discord server](https://discord.gg/eBQ7EfNZSZ).

**Comment by al0fdf** [comment, marked as answer]
Triplanar was set to `on` in the default material.
